packages/java/endpoint/src/main/java/dev/hilla/springnative/HillaHintsRegistrar.java

```diff
@@ -19,8 +30,13 @@
  */
 public class HillaHintsRegistrar implements RuntimeHintsRegistrar {
```

Comment:
Needs some tests.

I guess it would be hard to setup a native image integration test. How about a unit test for now though?

Reply:
Feel free to copy anything of mine you think is useful. I'm not sure what's needed here.

Reply:
Added a unit test for now